core/internal/cltest/factories.go

```diff
@@ -147,7 +147,24 @@ func NewJobWithFluxMonitorInitiator() models.JobSpec {
 		InitiatorParams: models.InitiatorParams{
 			Address:     NewAddress(),
 			RequestData: models.JSON{gjson.Parse(`{"data":{"coin":"ETH","market":"USD"}}`)},
-			Feeds:       []string{"https://lambda.staging.devnet.tools/bnc/call"},
+			Feeds:       models.JSON{gjson.Parse(`["https://lambda.staging.devnet.tools/bnc/call"]`)},
+			Threshold:   0.5,
+			Precision:   2,
+		},
+	}}
+	return j
+}
+
+// NewJobWithFluxMonitorInitiator create new Job with FluxMonitor initiator
+func NewJobWithFluxMonitorInitiatorWithBridge() models.JobSpec {
```

Comment:
We should def come up with a cleaner way for creating these Test Objects at some point.